Taskfile.yml

```
@@ -0,0 +1,27 @@
+version: '3'
+
+dotenv: [".task.env", ".env"]
+
+vars:
+  DOCKER_COMPOSE: '{{ .TASK_DOCKER_COMPOSE | default "itkdev-docker-compose-server" }}'
+
+tasks:
+  template-prune:
+    cmds:
+      - sh -c "echo 'COMPOSE_PROJECT_NAME={{.PROJECTNAME}}\nCOMPOSE_DOMAIN={{.PROJECTNAME}}.local.itkdev.dk' > .env"
+      - sh -c "echo 'TASK_DOCKER_COMPOSE=itkdev-docker-compose' > .task.env"
+      - sh -c "rm -R web/themes/custom/TEMPLATE_THEME"
+      - sh -c "echo '\$settings[\"trusted_host_patterns\"] = [\n  \"^{{.PROJECTNAME}}\.local\.itkdev\.dk$\"\n];' >> deploy-templates/settings.local.php"
+      - sh -c "mkdir -p web/sites/default"
+      - sh -c "cp deploy-templates/settings.local.php web/sites/default/"
+      - sh -c "cp deploy-templates/services.local.yml web/sites/default/"
+      - sh -c "cp deploy-templates/Taskfile.yml Taskfile.yml"
+      - sh -c "rm -R deploy-templates"
```

Task uses `sh`, so you don't need `ssh -c` (cf. https://taskfile.dev/usage/#:~:text=Task%20uses%20mvdan.cc/sh%2C%20a%20native%20Go%20sh%20interpreter):

```suggestion
      - echo 'COMPOSE_PROJECT_NAME={{.PROJECTNAME}}\nCOMPOSE_DOMAIN={{.PROJECTNAME}}.local.itkdev.dk' > .env
      - echo 'TASK_DOCKER_COMPOSE=itkdev-docker-compose' > .task.env
      - rm -R web/themes/custom/TEMPLATE_THEME
      - echo '\$settings[\"trusted_host_patterns\"] = [\n  \"^{{.PROJECTNAME}}\.local\.itkdev\.dk$\"\n];' >> deploy-templates/settings.local.php
      - mkdir -p web/sites/default
      - cp deploy-templates/settings.local.php web/sites/default/
      - cp deploy-templates/services.local.yml web/sites/default/
      - cp deploy-templates/Taskfile.yml Taskfile.yml
      - rm -R deploy-templates
```